packages/story-editor/src/components/panels/panel/shared/title.js

```diff
@@ -88,7 +88,6 @@ const IconWrapper = styled.div`
   position: relative;
   width: 32px;
   height: 32px;
-
```

Comment:
😕 what do you think about the spaces here? I like spaces for readability but it's not a big deal.